client/src/Components/Common/Header/Header.jsx

```diff
@@ -22,6 +26,10 @@ export default ({
         />
         <Logo showlogo={showlogo} />
       </Grid>
+      <Grid item xs={2} sm={1}>
+        <BurgerMenu showMeun={showMeun} isLogin={isLogin} isAdmin={isAdmin} />
```

Comment:
typo of menu here

Reply:
new idea  , it is more powerful from props